source/Calamari.Azure/Commands/HealthCheckAzureCloudServiceCommand.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using Calamari.Azure.Accounts;
+using Calamari.Commands.Support;
+using Calamari.Deployment;
+using Calamari.Integration.Certificates;
+using Calamari.Integration.Processes;
+using Microsoft.WindowsAzure.Management.Compute;
+
+namespace Calamari.Azure.Commands
+{
+    [Command("hc-azure-cs", Description = "Run a health check on an Azure Cloud Service")]
+    public class HealthCheckAzureCloudServiceCommand : Command
```

Comment:
Yeah, I like it!  Health check commands like this was what I was expecting for cloud workers.